src/main/scala/org/jetbrains/sbt/Extractor.scala

```diff
@@ -126,7 +126,16 @@ object Extractor extends ExtractorBase {
         managed.map(DirectoryData(_, managed = true)) ++ unmanaged.map(DirectoryData(_, managed = false))
       }
 
-      ConfigurationData(configuration.name, sources, resources, output)
+      val excludes = {
+        try {
+          val ideExcludePaths = SettingKey[Seq[File]]("ide-exclude-paths")
```

Comment:
I suggest to use "directories" instead of "paths" because SBT keys tend to do the same. Also "excluded" seems to be better than "exclude" (verb). So let it be "ide-excluded-directories".

The naming matters because it's a part of public API which supposed to be stable.